cluster/vsphere/config-common.sh

```diff
@@ -33,3 +33,5 @@ SSH_OPTS="-oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -oLogLevel=E
 # Set GOVC_INSECURE if the host in GOVC_URL is using a certificate that cannot
 # be verified (i.e. a self-signed certificate), but IS trusted.
 # export GOVC_INSECURE=1
+# export GOVC_REGION='region' # The region where the VM is deployed
```

Comment:
These cannot be mapped to VC terminology directly and is a function of what the deployment code wants for k8s. It can be implemented rack level, metro level etc.. I would recommend to keep this PR as is and merge it into k8s repo via a PR